### ISSUE TITLE
Unable to install modules in 1.8.102 - CSP error

### ISSUE THREAD

**Issue by bfren** [freescout-helpdesk/freescout#3445]
PHP version: 8.2.10
FreeScout version: 1.8.102

After updating I am unable to install modules, with a CSP error.  In the console I get this after entering a (valid) license key and clicking the 'Install Module' button:

```
Content-Security-Policy: The page's settings blocked the loading of a resource at inline ("script-src").
Source: javascript:installModule('ticketnumber')…
```

I have refreshed / restarted etc.

**Comment by bfren** [comment]
My meta tag is:

```
<meta http-equiv="Content-Security-Policy" content="script-src 'self' 'nonce-xxxxxx'  ">
```

I believe the error I see is because this doesn't include 'unsafe-inline' - so perhaps the script for the install module button needs to be in a separate js file, so unsafe-inline doesn't need to be enabled?

**Comment by freescout-helpdesk** [comment]
Fixed in FreeScout v1.8.103.
